FreeRTOS-Plus/Demo/AWS/Ota_Windows_Simulator/Common/subscription-manager/subscription_manager.c

```diff
@@ -0,0 +1,170 @@
+/*
```

Comment:
Are we copying this code in multiple demos?

Reply:
This is shared by the OTA demos and not with other demos in the repository

Reply:
If possible, yes. The idea is to avoid code duplication.

Reply:
Thats good point but  this will require updates in all the other demos as well, as this is close to release we can cover this in next releases.

Reply:
It would be nice with no code duplication, but i think we can refactor that after the release. Also it's a demo, user will wrap one of the demo to play with, so the duplication doesn't happen in a single project.